ModelReplacementAPI/BodyReplacementBase.cs

```diff
@@ -212,18 +212,14 @@ protected virtual void Update()
             // Local/Nonlocal renderer logic
             if (!renderLocalDebug)
             {
-
                 if (RenderBodyReplacement()) {
                     SetRenderers(true);
-                    controller.thisPlayerModel.enabled = false; // Don't render original body if non-local player
-                    controller.thisPlayerModelLOD1.enabled = false;
-                    controller.thisPlayerModelLOD2.enabled = false;
-                    nameTagObj.enabled = false;
-                    nameTagObj2.enabled = false;
+                    SetPlayerControllerRenderers(false); // Don't render original body if non-local player
                 }
                 else
                 {
                     SetRenderers(false); // Don't render model replacement if local player
+                    SetPlayerControllerRenderers(true);
```

Comment:
This is to render back the model when, for example, we change back from third person view. Before it would keep the model disabled so the mirror wouldn't display anything.